pallets/corporate-actions/src/ballot/mod.rs

```diff
@@ -342,7 +364,7 @@ decl_module! {
         /// - `NotTargetedByCA` if the CA does not target `origin`'s DID.
         /// - `InsufficientVotes` if the voting power used for any motion in `votes`
         ///    exceeds `origin`'s DID's voting power.
-        #[weight = 950_000_000]
+        #[weight = <T as Trait>::BallotWeightInfo::vote(votes.len() as u32, 1)]
```

Comment:
All of these constant arguments provide a fixed value for cases where the complexity of an extrinsic scales linearly or log-linearly with the size in storage rather than an input passed to the extrinsic. The parameter thus quantifies the size in storage and the benchmark acknowledges this.

We could fix the value inside the benchmark instead, but I prefer to bubble up the issue as high up as possible. The value `1` is just temporary; we should add limits as necessary later on, but I'd like to land this first.

Reply:
Let's capture this in Jira.

Reply:
https://polymath.atlassian.net/browse/MESH-1433